symfony/flex/1.0/.env

```diff
@@ -1,3 +1,3 @@
 # This file is a "template" of which env vars need to be defined for your application
-# Copy this file to .env file for development, create environment variables when deploying to production
+# Override the variabled you want in .env.local for development, create environment variables when deploying to production
```

Comment:
`variables`

Reply:
Also, this would only be correct with FWB > 4.2. I don't really know how, but ideally, we want this change to only be there for FWB > 4.2.

Reply:
Yes I don't know how to handle this properly for old versions. The only way I see is creating a Flex 2.0 version.

Reply:
I don't think so. And it doesn't help, you can have FWB 2.2 with Flex 1.0, or more likely Flex 1.1 with Symfony 3.4.